x-pack/solutions/observability/plugins/synthetics/docs/openapi/synthetic_apis.yaml

```
@@ -0,0 +1,1446 @@
+openapi: 3.0.1
+info:
+  title: Synthetics
+  description: OpenAPI schema for Synthetics endpoints
+  version: '1.0'
+  contact:
+    name: Actionable Observability Team
+  license:
+    name: Elastic License 2.0
+    url: https://www.elastic.co/licensing/elastic-license
+servers:
+  - url: https://{kibana_url}
+    variables:
+      kibana_url:
+        default: localhost:5601
+tags:
+  - name: synthetics
+paths:
+  /api/synthetics/monitors:
+    get:
+      summary: Get monitors
+      operationId: get-synthetic-monitors
+      description: >
+        Get a list of monitors.
+
+        You must have `read` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
+      tags:
+        - synthetics
+      parameters:
+        - name: filter
+          description: Additional filtering criteria.
+          in: query
+          schema:
+            type: string
+        - name: locations
+          description: The locations to filter by.
+          in: query
+          schema:
+            oneOf:
+              - type: string
+              - type: array
+        - name: monitorTypes
+          description: The monitor types to filter.
+          in: query
+          schema:
+            oneOf:
+              - type: string
+                enum:
+                  - browser
+                  - http
+                  - icmp
+                  - tcp
+              - type: array
+        - name: page 
+          description: The page number for paginated results.
+          in: query
+          schema:
+            type: integer
+        - name: per_page 
+          description: The number of items to return per page.
+          in: query
+          schema:
+            type: integer
+        - name: projects
+          description: The projects to filter by.
+          in: query
+          schema:
+            oneOf:
+              - type: string
+              - type: array
+        - name: query
+          description: A free-text query string
```

Just for consistency...

```suggestion
          description: A free-text query string.
```